pkg/resources/constants/subscriptionConstants.go

```diff
@@ -7,4 +7,5 @@ const (
 	GrafanaSubscriptionName       = "rhmi-grafana"
 	Marin3rSubscriptionName       = "rhmi-marin3r"
 	ObservabilitySubscriptionName = "rhmi-observability"
+	MCGSubscriptionName           = "rhmi-mcg"
```

Comment:
We can indeed, I opted for rhmi for consistency with all of the others but would be happy to update it to rhoam

Reply:
Or maybe integreatly-mcg? rather than going with a specific install type acronym. @MStokluska @briangallagher do you think it's worth doing this or rather just keep them?